test/integration/patients/patient/archive.js

```diff
@@ -317,69 +352,86 @@ context('patient archive page', function() {
-            details: null,
-            duration: 0,
-            due_date: null,
-            due_time: null,
-            updated_at: testTs(),
-          },
-          relationships: {
-            patient: { data: { id: '11111' } },
-            owner: {
-              data: {
-                id: '11111',
-                type: 'clinicians',
-              },
+        fx.data = [
+          getAction({
+            id: '1',
+            attributes: {
+              name: 'First In List',
+              details: null,
+              duration: 0,
+              due_date: null,
+              due_time: null,
+              updated_at: testTs(),
             },
-            state: { data: { id: '55555' } },
-            form: { data: { id: '11111' } },
-            files: { data: [{ id: '1' }] },
-          },
-        };
-
-        fx.data[1].attributes.name = 'Third In List';
-        fx.data[1].relationships.state = { data: { id: '55555' } };
-        fx.data[1].relationships.owner = { data: { id: '11111', type: 'teams' } };
-        fx.data[1].attributes.updated_at = testTsSubtract(2);
-        fx.data[1].attributes.due_time = '09:00:00';
-        fx.data[1].attributes.due_date = testDateSubtract(2);
+            relationships: {
+              patient: getRelationship('11111', 'patients'),
+              owner: getRelationship('11111', 'clinicians'),
+              state: getRelationship(stateDone),
+              form: getRelationship(testForm),
+              files: getRelationship([{ id: '1' }], 'files'),
+            },
+          }),
+          getAction({
+            attributes: {
+              name: 'Third In List',
+              updated_at: testTsSubtract(2),
+              due_time: '09:00:00',
+              due_date: testDateSubtract(2),
+            },
+            relationships: {
+              state: getRelationship(stateDone),
+              owner: getRelationship(teamCoordinator),
+            },
+          }),
+        ];
 
         return fx;
       })
       .routePatientFlows(fx => {
-        fx.data = _.sample(fx.data, 2);
-
-        fx.data[0].attributes.name = 'Second In List';
-        fx.data[0].relationships.state = { data: { id: '55555' } };
-        fx.data[0].relationships.owner = { data: { id: '11111', type: 'clinicians' } };
-        fx.data[0].attributes.updated_at = testTsSubtract(1);
-
-        fx.data[1].attributes.name = 'Last In List';
-        fx.data[1].id = '2';
-        fx.data[1].relationships.state = { data: { id: '55555' } };
-        fx.data[1].relationships.owner = { data: { id: '11111', type: 'teams' } };
-        fx.data[1].attributes.updated_at = testTsSubtract(6);
+        fx.data = [
+          getFlow({
+            attributes: {
+              name: 'Second In List',
+              updated_at: testTsSubtract(1),
+            },
+            relationships: {
+              state: getRelationship(stateDone),
+              owner: getRelationship('11111', 'clinicians'),
```

Comment:
I believe all of the `getRelationship('11111', 'clinicians')` could be

```js
const currentClinician = getCurrentClinician();

... getRelationship(currentClinician),
```

and the `11111` tests could look at `currentClinician.id`

Reply:
I took a crack at this in this commit: https://github.com/RoundingWell/care-ops-frontend/pull/1243/commits/dc624e02c295ae0810340622428d9b0c577864f8